all-of-us/workbench
# [DT-458][risk=no] update dev up tanagra to use service account

---
- Run tanagra server as a service account that uses `sa-key.json` for local deploy
- Tested with launcing UI from `workbench/tanagra/ui`
**Testing**
- set `enableTanagra=true` in `config-local.json`
- run `./project.rb dev-up`
- wait to complete then run `./project.rb dev-up-tanagra --disable-auth`
- cd to `/workbench/tanagra/ui` -> run `npm run codegen`, then `npm --force install` then `npm start`
   -  this should open `localhost:3000` in default browser and show the RT, CT indexes
- run `/workbench/ui: yarn dev-up-local`
- Launch test/local chrome browser `localhost:4200` dismiss any console errors 
- switch to `TANAGRA` tab
<img width="1205" alt="image" src="https://github.com/all-of-us/workbench/assets/4452480/0923be8e-e0aa-420b-ada1-373091fd75ce">


---
**PR checklist**

- One or more of the following is true:
